internal/tiltfile2/tiltfile_state.go

```diff
@@ -23,6 +23,7 @@ type tiltfileState struct {
 	// set at creation
 	ctx      context.Context
 	filename string
+	root     string
```

Comment:
hmmm...doesn't TiltfileState already have a function for this? it should be absPath or smth?

Reply:
We have functions for dealing with the paths for static (image) builds, but not one for the Tiltfile afaict. Got a link?

Reply:
Aha! it was in the `files` file.